test/conversion_test.go

```diff
@@ -1013,14 +986,6 @@ func TestPipelineRunCRDConversion(t *testing.T) {
 	}
 }
 
-func withFullEmbeddedStatus(ctx context.Context) context.Context {
-	featureFlags, _ := config.NewFeatureFlagsFromMap(map[string]string{
-		"embedded-status": config.FullEmbeddedStatus,
-	})
-	cfg := &config.Config{FeatureFlags: featureFlags}
-	return config.ToContext(context.Background(), cfg)
-}
-
 // validatePipelineRunTaskRunStatusAnnotations validates the `status.taskRun` in Annotations by converting
 // the json string into a struct and compare the valid status fields
 func validatePipelineRunTaskRunStatusAnnotations(pipelineRunName string, annotations map[string]string) error {
```

Comment:
Is this still needed?

Reply:
Thanks for spotting this. I hesitated to remove this since there could be still users using `taskruns` / `runs` for the time being? If so I could remove this in the PR that does the removal?